.github/workflows/finishReleaseCycle.yml

```diff
@@ -73,9 +73,63 @@ jobs:
         with:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
 
+  checkAndUpdateVersion:
+    name: Check and update version if production is greater than staging
```

Comment:
The GH issue comment is good, but in addition a comment in the code explaining in what cases this happens would be useful.